Client/Components/DashBoard.razor.cs

```diff
@@ -51,35 +45,39 @@ protected override async Task OnInitializedAsync()
 
         private async void HomeCountryChange(string CountryCode)
```

Comment:
Can this method be removed? As it only contains commented code.

Reply:
Can't be removed at the moment, since parts of the ui use the method. The code is for doing the manual selection of a country if it can't be detected. It should probably be refactored to use this state management system, but I do not think it is a part of this setup pr.